modules/installation-vsphere-config-yaml.adoc

```diff
@@ -153,13 +153,21 @@ value must match the number of control plane machines that you deploy.
 <6> The cluster name that you specified in your DNS records.
 <7> Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
 <8> The vSphere datacenter.
-<9> Optional parameter. For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
-<10> Optional parameter For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass` object, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
-<11> The fully-qualified hostname or IP address of the vCenter server.
+<9> The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
++
+[IMPORTANT]
+====
+You can specify the path of any datastore that exists in a datastore cluster. By default, Storage vMotion is automatically enabled for a datastore cluster. Red Hat does not support Storage vMotion, so you must disable Storage vMotion to avoid data loss issues for your {product-title} cluster.
```

Comment:
same as above

Storage vMotion is not enabled or disabled on a datastore cluster, Storage DRS is.

Reply:
Thanks, @jcpowermac . Updated. You OK to add a LGTM?